Assets/_BForBoss/_Character/Scripts/DashLinesEffectBehaviour.cs

```diff
@@ -38,11 +37,10 @@ private void Update()
             var isLerpingToZero = _currentTime <= 0;
             var resultingAlpha = isLerpingToZero ? 0 : _maxAlpha;
 
-            _speedLineMaterial.SetFloat(MATERIAL_ALPHA, Mathf.Lerp(currentAlpha, resultingAlpha, _lerpInStrength));
-            
+            _speedLineMaterial.SetFloat(MATERIAL_ALPHA, Mathf.Lerp(currentAlpha, resultingAlpha, _lerpStrength));
+
             _currentTime -= Time.deltaTime;
-            
-            
+
```

Comment:
There's a `Debug.Log(_currentTime);` in the Play section, can u remove it.

Reply:
Removed!

Reply:
Removed!